### ISSUE TITLE
using MutableArithmetic api?

### ISSUE THREAD

**Issue by kalmarek** [kalmarek/Arblib.jl#118]
so that we can write `MA.@rewrite a + b + c` instead of sequence of `Arblib.add!`s.

**Comment by Joel-Dahne** [comment]
I have looked a little bit at this package before and I think it would be very nice to support it! It's a perfect fit for our use case. I would be happy to take a look at this at some point but at the moment I'm focused on, slowly, improving the support for polynomials and series (which would both benefit even more from this than for example `Arb` would).

**Comment by kalmarek** [comment]
yeah, this was more of a reminder for us, @saschatimme spearheaded the package so he'll be more qualified to implement this ;)

But it's great to hear that you're still working on the package, I've been mostly taken by my other projects for quite a while!
